setup.py

```diff
@@ -58,13 +68,8 @@ def read(*parts):
     packages=find_packages('src'),
     package_dir={'': 'src'},
     include_package_data=True,
-    tests_require=test_requires,
-    install_requires=install_requires,
-    extras_require={
-        'docs': ['sphinx'],
-        'tests': test_requires,
-        'dev': dev_requires,
-    },
```

Comment:
Is there any harm in keeping the `extras_require` entry?

Reply:
there was just one dependency for docs i moved it to another file.